### PR TITLE
Android cross compilation

### PR DESCRIPTION
**Pull request by talklittle**
Part of solution for https://github.com/servo/servo/issues/3030

---
This change is [Reviewable](https://reviewable.io/reviews/servo/fontsan/9)

### CODE REVIEW

File: scripts/build-ots.sh [hunk 1]
```diff
@@ -46,7 +49,18 @@ else
     FLAGS="$FLAGS -O2"
 fi
 
-./configure --with-zlib="$fake_zlib" --host="$TARGET" \
+if [ -n "$DEP_FREETYPE_OUTDIR" ]; then
+    FREETYPE_CFLAGS="-I $DEP_FREETYPE_OUTDIR/include"
```
Comment (mbrubeck) [inline scripts/build-ots.sh]: May need extra quoting.

**Comment by mbrubeck** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/fontsan/9)*

**Comment by mbrubeck** [comment]
testing

<details><summary><i>Previously, bors-servo wrote…</i></summary>
> :pushpin: Commit deba44d has been approved by `mbrubeck`
> 
> 
</details>


---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/fontsan/9#-:-KJbXEm-h0grgRJq8jKD:726359403)*

**Comment by talklittle** [comment]
@jdm Do you have a special trick to get @bors-servo to test and merge PRs on this fontsan repo?

**Comment by jdm** [comment]
I do not. I'm not sure what's up.

**Pull request merged**